pocs/utils/google/storage.py

```diff
@@ -51,10 +51,11 @@ def __init__(self, bucket_name, auth_key=None, project_id='panoptes-survey'):
 
         try:
             self.bucket = self.client.get_bucket(bucket_name)
-        except Forbidden as e:
+        except Forbidden:
             raise error.GoogleCloudError(
-                "Storage bucket does not exist or no permissions. " +
-                "Ensure that the PANOPTES_CLOUD_KEY variable is properly set"
+                "Storage bucket does not exist or no permissions. "
+                "Ensure that the PANOPTES_CLOUD_KEY variable is properly set "
```

Comment:
This is removed in #577 as we don't use this env var anymore. Like above, I can either merge after or if you want to remove here.